### DISCUSSION TITLE
I need some clarification regarding the APIs

### DISCUSSION THREAD

**Discussion by volm92** [InterDigitalInc/AdvantEDGE#277] [Q&A] [answered]
Hello, 
I would like to change the characteristics of the network via API. From this [link](https://github.com/InterDigitalInc/AdvantEDGE/blob/master/docs/api-sandbox-ctrl/Apis/EventsApi.md#sendevent) I tried to understand how to send the request. I couldn't figure out what kind of payload the server is expecting. JSON?
Can you give me an example of a request to change the characteristics of the network? Thank you very much!

**Comment by roymx** [comment, marked as answer]
Hello @volm92 ,

You identified the correct API.

The easiest way to experiment with the API is using the Swagger client.

1- Open a web page tot he following URL:
`http://<your-platform-ip>/<your-sandbox-name>/api`

2- Select the Sandbox Controller API from the drop down list

3- you will see this when you open the EVENTS endpoint
![image](https://user-images.githubusercontent.com/48520190/125814155-d132d24b-4ab8-4066-97e9-7f0e8a8e1ea6.png)

4- the model is described there and should be self-explanatory

5- If you hit `try-it out` and then `execute` you will be able to send your event directly from the page
Note: this will also give you the curl command and the JSON object.

Here's an example
![image](https://user-images.githubusercontent.com/48520190/125851552-24464ab3-e822-4ae0-bca0-1b084bff9359.png)

**Reply by volm92** [reply]
Thankyou! 
Is there this [API](https://github.com/InterDigitalInc/AdvantEDGE/blob/master/docs/api-rnis/Apis/RniApi.md#layer2MeasInfoGET) in the swagger?

https://github.com/InterDigitalInc/AdvantEDGE/blob/master/docs/api-rnis/Apis/RniApi.md#layer2MeasInfoGET

Is this API working?

I tried it but it returns just timestamp. Maybe I'm wrong in doing something.
Thanks!

**Reply by roymx** [reply]
yes, it's there & working

![image](https://user-images.githubusercontent.com/48520190/126186918-05cfb62a-7957-4461-b48f-57f703bb6210.png)
